src/partials/config.html

```diff
@@ -1,12 +1,30 @@
 <h3 class="page-heading">OCI settings</h3>
 
 <div class="gf-form-group">
-<div class="gf-form max-width-30">
-  <label class="gf-form-label width-10">Tenancy OCID</label>
+  <div class="gf-form max-width-30">
+    <label class="gf-form-label width-10">Environment</label>
```

Comment:
What happens when an existing user with Grafana running and the OCI metrics plugin already installed and configured, upgrades to this version of the plugin?

Reply:
In particular, how will the new configuration fields be set? Will the plugin still be operational?

Reply:
Yes, it should work without modification. Some dashboards would need a data refresh to show the graphs.